Foundation/Summit/2024/social-activities.html

```diff
@@ -0,0 +1,127 @@
+---
+#    Copyright 2024 seL4 Project a Series of LF Projects, LLC.
+#    SPDX-License-Identifier: CC-BY-SA-4.0
+title: Summit program
+---
+
+<h1>
+  seL4 Summit 2024
+</h1>
+<h4>
+  Sydney, Australia
+</h4>
+<h4>
+  15-17 October 2024
+</h4>
+
+<h3>
+   Social Activities
+</h3>
+<p>
+    A number of informal <a href="social-activities">social activities</a> are being organised by local Sydney-siders. Check out the list and contact the organiser to register your interest.
+</p>
+<p>
+    This list is in chronological order. Note that some activities require advanced bookings, by 20 September 2024.
+</p>
+
+<div>
+    <h4 style="padding-top:20px">Blue Mountains bush walk (TBC)</h4>
+    <p>
+        Gernot plans to take a group on a walk, most likely in the stunning <a href="https://www.visitnsw.com/destinations/blue-mountains">Blue Mountains</a>, which are about a 1.5 hours drive from Sydney. The details of this activity will be organised mid-September 2024. Stay tuned!
+    </p>
+    <p>
+        <strong>When: </strong> Weekend of 12 & 13 October 2024 (TBC)
+    </p>
+    <p>
+        <strong>Contact: </strong><a href="https://www.unsw.edu.au/staff/gernot-heiser">Gernot Heiser</a>
+    </p>
+</div>
+
+<div>
+    <h4 style="padding-top:20px">Bondi to Coogee walk</h4>
+    <img src="../../../images/summit/bondi.jpg" alt="Bondi Beach" width="250" float="right">
+    <p>
+        June will take a group on the iconic <a href="https://www.sydney.com/things-to-do/nature-and-parks/walks/bondi-to-coogee-coastal-walk">Bondi to Coogee walk</a>, with dinner to follow at Coogee.
+    </p>
+    <p>
+        <strong>When: </strong> Monday 14 Oct 2024, starting at 3:30pm
+    </p>
+    <p>
+        <strong>Where: </strong><a href="https://maps.app.goo.gl/fDrRZqj7JHNGiFYx9">Bondi Beach</a>
+    </p>
+    <p>
+        <strong>Cost: </strong>Free, with the option of buying dinner at Coogee
+    </p>
+    <p>
+        Bring:
+    </p>
+    <ul>
+        <li>Hat</li>
+        <li>Sunscreen</li>
+        <li>Water bottle</li>
+        <li>Umbrella or raincoat</li>
+        <li>Swimmers and towel (optional)</li>
+    </ul>
+    <p>
+        <strong>Contact: </strong><a title="june@sel4.systems" href="mailto:june@sel4.systems">June Andronick</a>.
+    </p>
+</div>
+
+<div>
+    <h4 style="padding-top:20px">A guided tour of The Rocks</h4>
+    <p>
+        Birg will organise a guided tour of <a href="https://www.rockswalkingtours.com.au/walking-tour/">The Rocks</a>, which gives an account of Sydney's colonial heritage.
+    </p>
+    <p>
+        Bookings are required by <span class="highlight">20 September 2024</span>. Max group size 20.
```

Comment:
You can have an earlier date if needed. Or mention that sending early interest is desired to confirm the booking.